README.md

```diff
@@ -1,25 +1,41 @@
-# Backend Starter repository
+# Braze Content Proxy API
 
-This repository can be used as a template to spin up a new service. 
+Provides Pocket Hits stories to be consumed by Connected Content blocks in Braze.
 
-Note: please change all reference to `Acme` to your service name. There are `Todo`s in this repository
-that points to name changes to your new service name, please address them before spinning up the service.
+## Application Overview
 
-## Folder structure
-- the infrastructure code is present in `.aws`
-- the application code is in `src`
-- `.docker` contains local setup
-- `.circleci` contains circleCI setup
+[Express](https://expressjs.com/) is the Node framework, [Apollo Client](https://www.apollographql.com/docs/react/) is used in Express to request data from [Client API](https://github.com/Pocket/client-api/).
+
+## Caching
+
+### API-side caching
+
+TBA
```

Comment:
Leaving as "to be added" here as the caching we need is yet to be determined in https://getpocket.atlassian.net/browse/BACK-1407.